addons/scoreboard/mergedplayer.lua

```diff
@@ -234,8 +234,21 @@ function MergedPlayer:wsavg()
     end
 end
 
-
-
+function MergedPlayer:wsacc()
+    local hits, misses = 0, 0
+    
+    for _, p in ipairs(self.players) do
```

Comment:
This seemed strange to me as well, but it turns out that scoreboard stores one player instance in the database for each type of mob fought (see comments [here](https://github.com/Windower/Lua/blob/dev/addons/scoreboard/player.lua#L4)). So if you fought crabs and bats you would have two player instances per actual player.